gradle/libs.versions.toml

```diff
@@ -12,11 +13,15 @@ system-lambda = "1.2.1"
 micronaut = "3.4.3"
 micronaut-test = "3.4.0"
 
+# Testing
+
+testcontainers = "1.17.3"
```

Comment:
You don't need this version, it's part of Micronaut BOM. Which, by the way, you could take the opportunity to use latest 3.6.x (line 13)

Reply:
After updating micronaut to 3.6.x now azure-function-http-test tests are failing. Since this is not the scope of this PR I would rather go back to the prev version.

Reply:
And 3.5.x?

Reply:
Latest version that works is 3.5.1 so I guess we can use that. In 3.5.2 we get error like this 
```
Distributed Test Run :azure-function-http-test:test > Partition 2 in session 1 on localhost-executor-1 > AzureFunctionCorsSpec test preflight request with controlled headers FAILED

  io.micronaut.http.client.exceptions.HttpClientResponseException: Method Not Allowed
      at app//io.micronaut.http.client.netty.DefaultHttpClient$FullHttpResponseHandler.makeErrorFromRequestBody(DefaultHttpClient.java:3317)
```

Reply:
Weird. 3.5.1 is fine for the moment.